### DISCUSSION TITLE
Problem with alignment

### DISCUSSION THREAD

**Discussion by Ash100** [mdtraj/mdtraj#1821] [General]
Hello Everyone, 
I am learning by running some tutorials. I am supposed to run `results = align(complexes, method=METHODS["mda"])`, where complexes are PDBs structures with ligands. I am having the following error;

AttributeError                            Traceback (most recent call last)
[<ipython-input-76-75600e02e635>](https://localhost:8080/#) in <cell line: 1>()
----> 1 results = align(complexes, method=METHODS["mda"])

4 frames
[/usr/local/lib/python3.10/dist-packages/opencadd/structure/superposition/sequences.py](https://localhost:8080/#) in fasta2select(fastafilename, ref_resids, ref_segids, target_resids, target_segids, backbone_selection)
    113 
    114     """
--> 115     protein_gapped = Bio.Alphabet.Gapped(Bio.Alphabet.IUPAC.protein)
    116     with open(fastafilename) as fasta:
    117         alignment = Bio.AlignIO.read(fasta, "fasta", alphabet=protein_gapped)

AttributeError: module 'Bio' has no attribute 'Alphabet'

Would you please guide me, how to address this issue of alphabet?

**Comment by sukritsingh** [comment]
It looks like the issue you're having is stemming from something within the [OpenCadd](https://github.com/volkamerlab/opencadd) library? MDTraj doesn't support `fasta` at present.

Perhaps you'll find an answer opening an issue there?
